docs/en/openmldb_quickstart.md

```diff
@@ -0,0 +1,379 @@
+  c1 c2 c3 c4 c5 c6 c7
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+  aaa 12 22 2.200000 12.300000 1636097890000 1970-01-01
+  aaa 11 22 1.200000 11.300000 1636097290000 1970-01-01
+  dd 18 22 8.200000 18.300000 1636111690000 1970-01-01
+  aa 13 22 3.200000 13.300000 1636098490000 1970-01-01
+  cc 17 22 7.200000 17.300000 1636108090000 1970-01-01
+  ff 20 22 9.200000 19.300000 1636270090000 1970-01-01
+  bb 16 22 6.200000 16.300000 1636104490000 1970-01-01
+  bb 15 22 5.200000 15.300000 1636100890000 1970-01-01
+  bb 14 22 4.200000 14.300000 1636099090000 1970-01-01
+  ee 19 22 9.200000 19.300000 1636183690000 1970-01-01
+ ----- ---- ---- ---------- ----------- --------------- - -------------
+```
+
+:bulb: Notice:
+
+- Different from the stand-alone version of OpenMLDB, the cluster version of OpenMLDB needs to maintain offline and online data separately.
+- Users need to successfully complete the online deployment of SQL before preparing to go online. Otherwise, the online data may fail.
+
+#### 3.3.6 Exit the CLI
+
+```sql
+> quit;
+```
+
+Up to this point, you have completed all the development and deployment work based on the cluster version of OpenMLDB CLI, and have returned to the operating system command line.
+
+#### 3.3.7 Real-time feature extraction
+
+Notice: warning:: According to the default deployment configuration, the http port for apiserver deployment is 9080.
+
+Real-time online services can be provided through the following Web APIs:
+
+```
+http://127.0.0.1:9080/dbs/demo_db/deployments/demo_data_service
+        \___________/ \____/ \_____________/
+              | | |
+        APIServer address Database name Deployment name
+```
+
+The input data of the real-time request accepts the `json` format, and we put a line of data into the `input` field of the request. The following example:
+
+```bash
+curl http://127.0.0.1:9080/dbs/demo_db/deployments/demo_data_service -X POST -d'{"input": [["aaa", 11, 22, 1.2, 1.3, 1635247427000, "2021-05- 20"]]}'
+```
+
+The following is the expected return result for this query (the computed features are stored in the `data` field):
+
+```json
+{"code":0,"msg":"ok","data":{"data":[["aaa",11,22]]}}
+
+```
+
+#### 3.3.8 Result description of real-time feature extraction
+
+Real-time request (deployment execution) is SQL execution in request mode (request mode). Unlike batch mode (batch mode), request mode will only perform SQL extractions on request rows. In the previous example, the POST input is used as the request line, assuming this line of data exists in the table demo_table1, and execute SQL on it:
+```
+SELECT c1, c2, sum(c3) OVER w1 AS w1_c3_sum FROM demo_table1 WINDOW w1 AS (PARTITION BY demo_table1.c1 ORDER BY demo_table1.c6 ROWS BETWEEN 2 PRECEDING AND CURRENT ROW);
+```
+The specific extraction logic is as follows (the actual extraction will be optimized to reduce the amount of extraction):
```

Comment:
reduce the amount of extraction -> reduce the computation

Reply:
ok, thank you.